File: solutions/2427-first-letter-to-appear-twice/solution.py
class Solution(object):
    def repeatedCharacter(self, s):
        """
        :type s: str
        :rtype: str
        """
        st=set()
        b=0
        for i in s:
            if i not in st:
                st.add(i)
            else:
                return i
        

File: solutions/2427-first-letter-to-appear-twice/solution.java
class Solution {
    public char repeatedCharacter(String s) {
        List<Character> ls=new ArrayList<>();
        for(char a:s.toCharArray())
        {
            if(ls.contains(a))
                return a;
            ls.add(a);
        }
        return 'a';
    }
}
